nomos-services/storage/src/backends/mod.rs

```diff
@@ -1,10 +1,11 @@
 #[cfg(feature = "mock")]
 pub mod mock;
-#[cfg(feature = "sled")]
-pub mod sled;
-
 #[cfg(feature = "rocksdb")]
 pub mod rocksdb;
+#[cfg(feature = "sled")]
+pub mod sled;
+#[cfg(feature = "surrealdb-backend")]
```

Comment:
for consistency it could be just `"surrealdb"`

Reply:
That's fair. That being said, why use the library name instead of the feature name?

Reply:
But you named the feature. All the other backends use the same feature name as the library. Maybe I do not get your question though.

Reply:
Oh, I meant: Is there a purpose in using the library name for the feature name?E.g.: `sled` for the feature, instead of and `sled-backend`

Reply:
Probably because its shorter. If you want to use the `-backend` suffix then better to rename all the other features.

Reply:
Oh, okay, I thought there might be some special behaviour, or that it might interfere somehow. I'll stick to standards and go with just `surrealdb` then.